fastlane/Fastfile

```diff
@@ -356,6 +361,7 @@ platform :ios do
       entitlements_file_path: entitlements_file_path
     )
 
+    if (target_extension || bundle_identifier_extension || profile_name_extension || entitlements_file_path_extension)
```

Comment:
[Suggestion] I think by using OR (`||`), the statement in the `if` will be true even if some of the options are missing. Have you tried for example to pass to the lane only `target_extension` to see whether the `if` body is executed?

I am pretty sure that AND (`&&`) should be used, but Ruby is special... 🤔

Reply:
and I was thinking it will fail if only `target_extension` is passed and you need to fix it by passing the other parameters? 🤔